packages/nvidia_nat_phoenix/src/nat/plugins/phoenix/scripts/export_trajectory_to_phoenix/export_atif_trajectory_to_phoenix.py

```
@@ -0,0 +1,109 @@
+
+        logging.info(
+            "Exporting %s  (agent=%s, steps=%d, session=%s)",
+            path.name,
+            agent_name,
+            num_steps,
+            session_id,
+        )
+
+        try:
+            exporter.export(trajectory)
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)